web/scripts/generateZonesConfig.ts

```diff
@@ -37,6 +37,7 @@ const getConfig = (): CombinedZonesConfig => {
     'parsers',
     'subZoneNames',
     'aggregates_displayed',
+    'generation_only',
```

Comment:
Instead of including this would it not make more sense to exclude all the zone configs that contain this?

And do any of these zones have exchanges we somehow need to account for that we might not have considered yet?

Reply:
Hmm, since we still want to allow accessing via URL I don't think we can just do that? 🤔

Reply:
Wouldn't that be very weird if we didn't display it on the map?

Like if someone shared a screenshot or mentioned it no other users would be able to find it unless they edited the URL (which is impossible if you use the App Store versions of the app).

Reply:
We have the same with old India zones: https://app.electricitymaps.com/zone/IN-MH

I really don't think that will be a big problem and it's easier for debugging to see the zone on the app. Happy to reconsider if we ever reach a situation where this becomes a problem :)

Reply:
Didn't we delete those? 😅 🙈

Reply:
Nope, not yet - too much work since internal tests are relying on it 😬

Reply:
Ah well, let's just use your current approach then and we can revisit when we delete the old India zones (and exchanges).